packages/block-editor/src/hooks/anchor.js

```diff
@@ -71,7 +71,18 @@ export const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) =>
 					<InspectorAdvancedControls>
 						<TextControl
 							label={ __( 'HTML Anchor' ) }
-							help={ __( 'Anchors lets you link directly to a section on a page.' ) }
+							help={ (
+								<>
+									<p>
+										{ __( 'Enter a word or two — without spaces — to make a unique web address just for this heading, called an "anchor." Then, you\'ll be able link directly to this section of your page.' ) }
+									</p>
+									<p className="components-base-control__learn-more">
+										<ExternalLink href={ __( 'https://wordpress.org/support/article/#/' ) }>
```

Comment:
This URL doesn't go anywhere useful.

Reply:
We need to sub it in for https://wordpress.org/support/article/page-jumps/ now that the page exists